### PR TITLE
Debian 8 support for postgresql & rabbitmq

### PR DESCRIPTION
**Pull request by vuntz**

### CODE REVIEW

File: chef/cookbooks/postgresql/recipes/apt_pgdg_postgresql.rb [hunk 1]
```diff
@@ -1,4 +1,4 @@
-if not %w(squeeze wheezy sid lucid precise saucy trusty).include? node["postgresql"]["pgdg"]["release_apt_codename"]
+if not %w(jessie squeeze wheezy sid lucid precise saucy trusty).include? node["postgresql"]["pgdg"]["release_apt_codename"]
```
Comment (vuntz) [inline chef/cookbooks/postgresql/recipes/apt_pgdg_postgresql.rb]: I prefer to not fix these to avoid diverging too much from upstream cookbook.

**Comment by vuntz** [comment]
Anyone up for a review of this?

**Comment by rsalevsky** [comment]
LGTM

**Comment by toabctl** [comment]
+1 (assuming you tested it)

**Comment by toabctl** [comment]
Still +1.

**Comment by rsalevsky** [comment]
+1

**Comment by mjura** [comment]
+1

**Pull request merged**